python/pyspark/sql/column.py

```diff
@@ -507,6 +507,24 @@ def __iter__(self):
     >>> df.filter(df.name.like('Al%')).collect()
     [Row(age=2, name='Alice')]
     """
+    _ilike_doc = """
+    SQL ILIKE expression (case insensitive LIKE). Returns a boolean :class:`Column`
+    based on a case insensitive match.
+
```

Comment:
I know it is consistent with other places in the docs but let's add versionadded directive. I think we should add them actually for all places.

Reply:
Thank you, I'm also concerned about the versionadded directive... I will add it.